startup/BMM/user_ns/detectors.py

```diff
@@ -18,7 +18,7 @@
 
 run_report('\t'+'dwelltime')
 with_pilatus = False
-with_quadem, with_struck, with_dualem, with_xspress3 = True, True, False, False
+with_quadem, with_struck, with_dualem, with_xspress3 = True, True, False, True
```

Comment:
I believe that change uncovered the previously untested cases with xspress3, that's why the CI is failing.

Reply:
Agreed.